src/table/components/TableWrapper.jsx

```diff
@@ -1,19 +1,21 @@
 import PropTypes from 'prop-types';
-import React, { useEffect, useMemo, useRef, useCallback } from 'react';
+import React, { useEffect, useRef, useCallback } from 'react';
 import Table from '@mui/material/Table';
 
 import AnnounceElements from './AnnounceElements';
 import TableBodyWrapper from './TableBodyWrapper';
 import TableHeadWrapper from './TableHeadWrapper';
 import FooterWrapper from './FooterWrapper';
-import PaginationContent from './PaginationContent';
-import useDidUpdateEffect from './useDidUpdateEffect';
 import { useContextSelector, TableContext } from '../context';
 import { StyledTableContainer, StyledTableWrapper } from '../styles';
+
+import PaginationContent from './PaginationContent';
+import useDidUpdateEffect from '../hooks/use-did-update-effect';
+import useFocusListener from '../hooks/use-focus-listener';
+import useScrollListener from '../hooks/use-scroll-listener';
 import { handleTableWrapperKeyDown } from '../utils/handle-key-press';
-import { updateFocus, handleResetFocus, handleFocusoutEvent, getCellElement } from '../utils/handle-accessibility';
-import { handleHorizontalScroll, handleNavigateTop } from '../utils/handle-scroll';
-import announcementFactory from '../utils/announcement-factory';
```

Comment:
the file announcement-factory.ts in utils can be deleted, right?

Reply:
done